src/process-mining-visualisation/MiningView.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  svg,
+  RenderingContext,
+  Point,
+  EdgePadding,
+  IViewArgs,
+  IntersectingRoutedPoint,
+  PolylineEdgeViewWithGapsOnIntersections,
+  SEdgeImpl,
+  angleOfPoint,
+  toDegrees
+} from '@eclipse-glsp/client';
+import { VNode } from 'snabbdom';
+import { injectable } from 'inversify';
+import { Edge, EdgeLabel } from '@axonivy/process-editor';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const JSX = { createElement: svg };
+
+@injectable()
+export class MiningView extends PolylineEdgeViewWithGapsOnIntersections {
+  protected renderLine(edge: Edge, segments: Point[], context: RenderingContext): VNode {
+    if (edge.args && edge.args['labelvalue'] !== undefined && edge.children.length > 0) {
```

Comment:
For this args stuff, there is a utility form glsp called `GArgument`.
So e.g. you can do something like `GArgmuent.getString(edge, 'labelvalue')` and it does all the checks if the element has args etc.
So I think you could simplify your code a bit with this.